.github/workflows/CI-commercial-core.yml

```diff
@@ -121,8 +128,11 @@ jobs:
           path: dist
 
       - name: Release
+        uses: cycjimmy/semantic-release-action@v2
```

Comment:
Would it be preferable to use the GHA from semantic-release themselves (assuming it does the same thing):

https://github.com/semantic-release/github

Reply:
I don't think that's a GHA you've linked? And I can't find an official one, TBH I can't see why I switched to the action from just running the command! I'll switch it back..